lib/smart_answer_flows/uk-benefits-abroad/questions/is_either_of_the_following.govspeak.erb

```diff
@@ -1,14 +1,13 @@
 <% content_for :title do %>
-  Are you getting Income Support while either:
+  Which of these applies to you?
 <% end %>
 
-<% options(
-  "yes": "Yes",
-  "no": "No"
-) %>
-
-<% content_for :body do %>
-  - getting [Statutory Sick Pay](/statutory-sick-pay/)
-  - incapable of work, but being treated as capable of work because you are temporarily disqualified from receiving Income Support
+<% content_for :hint do %>
+  If neither apply, click 'Next Step'.
 
 <% end %>
+
+<% options(
+  "too_ill_to_work": "I’m getting Statutory Sick Pay",
```

Comment:
I had a meeting with accessibility specialist and content designer and we agreed verbally to remove the link.